etrace/Options.cs

```diff
@@ -120,18 +121,239 @@ public void PostParse()
+                    result = ParseAnyOfFilter(filter, Filter.LESS_OR_EQUAL_SIGN);
+                if (filter.Contains(Filter.GREATER_SIGN))
+                    result = ParseAnyOfFilter(filter, Filter.GREATER_SIGN);
+                else if (filter.Contains(Filter.LESS_SIGN))
+                    result = ParseAnyOfFilter(filter, Filter.LESS_SIGN);
+                else if (filter.Contains(Filter.NOT_EQUAL_SIGN))
+                    result = ParseAnyOfFilter(filter, Filter.NOT_EQUAL_SIGN);
+                else
+                    result = ParseAnyOfFilter(filter, Filter.EQUAL_SIGN);
+            }
+
+            return result;
+        }
+
+        private string[] SplitMultipleFilter(string filter, string splitby)
+        {
+            filter = filter.Replace(" ", "");
+            string[] result = filter.Split(new string[] { splitby }, StringSplitOptions.RemoveEmptyEntries);
+
+            if (result.Length == 1)
+                throw new ArgumentException($"Invalid filter: {filter}");
+
+            return result;
+        }
+
+        private string[] SplitBinaryFilter(string filter, string splitby)
+        {
+            filter = filter.Replace(" ", "");
+            string[] result = filter.Split(new string[] { splitby }, StringSplitOptions.RemoveEmptyEntries);
+
+            if (result.Length != 2)
+                throw new ArgumentException($"Invalid filter: {filter}");
+
+            return result;
+        }
+
+        private Filter ParseAnyOfFilter(string filter, string @operator)
+        {
+            var splited = SplitBinaryFilter(filter, @operator);
+            return new Filter(splited[0], splited[1], @operator);
+        }
+
+        #endregion
+
         // TODO LINQ-like filter, e.g. `e["Duration"] > 1000 && e.ProcessName == "notepad"`,
         //      potentially also with `select new { ... }` which is what we print
 
         // TODO Something that operates on event pairs, Event/Start and Event/Stop, and then
         //      allows queries on duration
 
-        // TODO Filters with greater-than or less-than operators
-
-        public Dictionary<string, Regex> ParsedFilters { get; } = new Dictionary<string, Regex>();
+        public List<Filter> ParsedFilters { get; } = new List<Filter>();
         public Regex ParsedRawFilter { get; private set; }
         public bool IsFileSession => !String.IsNullOrEmpty(File);
         public long ParsedClrKeywords { get; private set; } = 0;
         public KernelTraceEventParser.Keywords ParsedKernelKeywords { get; private set; } = KernelTraceEventParser.Keywords.None;
+
+
+        public class MultipleFilter : Filter
+        {
+            internal const string MULTIPLE_FILTER_SIGN = "&&";
+
+            internal MultipleFilter() : base ()
+            {
+                SubFilters = new List<Filter>();
+            }
+
+            public List<Filter> SubFilters { get; set; }
+
+            /// <summary>
```

Comment:
Feels a bit weird that only this method has xmldoc. I'd remove the comment, or make it a regular comment.